src/components/Input/Input.tsx

```diff
@@ -74,22 +71,15 @@ export default function Input(props: InputProps) {
         value: value,
         "aria-label": ariaLabel,
         "aria-labelledby": ariaLabelledBy,
+        disabled: disabled,
     };
 
     if (required) {
         inputProps["aria-required"] = true;
     }
 
-    if (labelId && !helperTextId) {
-        inputProps["aria-labelledby"] = labelId;
-    } else if (helperTextId && !labelId) {
-        inputProps["aria-labelledby"] = helperTextId;
-    } else if (labelId && helperTextId) {
-        inputProps["aria-labelledby"] = labelId + " " + helperTextId;
-    }
-
     let transformedInput = (
-        <input id={`input-${id}`} {...inputProps} placeholder={placeholder} />
+        <input id={id} {...inputProps} placeholder={placeholder} />
```

Comment:
are you adding attributes to this here or in the attributes PR? (I don't see it in the attributes pr)

Reply:
Whoops! Attributes PR. I've pushed it up over there.